types/ramda/tools.d.ts

```diff
@@ -408,6 +408,24 @@ export type ObjPred<T = unknown> = (value: any, key: unknown extends T ? string
  */
 export type Ord = number | string | boolean | Date;
 
+/*
```

Comment:
Please use [jsdoc style comment](https://google.github.io/styleguide/tsguide.html#jsdoc-vs-comments).

Can you modify other comments introduced before to jsdoc style comment?